### PR TITLE
Dev/commands+ui

### PR DESCRIPTION
**Pull request by IoannisKl**
Bar no longer empties, while the time ticks down. Also commands /bossbar enable, /bossbar disable, /bossbar TextsReload are implemented.

### CODE REVIEW

File: .idea/jarRepositories.xml [hunk 1]
```diff
@@ -1,6 +1,11 @@
 <?xml version="1.0" encoding="UTF-8"?>
```
Comment (EriolEandur) [inline .idea/jarRepositories.xml]: I prefer to not have .idea config files in the repository. 
I have troubles building the project in these cases quite often.

File: src/main/java/me/John/BossBanner/Main.java [hunk 2]
```diff
@@ -85,7 +85,7 @@ public boolean onCommand(CommandSender sender, Command cmd, String label, String
 			if (args[0].equalsIgnoreCase("TextsReload")) {
 				if (sender instanceof Player) {//player command
 					Player player = (Player) sender;
-					if (player.hasPermission("bossbar.admin")) {//has permission
+					if (player.hasPermission("admin")) {//has permission
```
Comment (EriolEandur) [inline src/main/java/me/John/BossBanner/Main.java]: permission strings should include the plugin name: bossbanner.admin
Also permissions should be declared in plugin.yml file
https://bukkit.gamepedia.com/Plugin_YAML

**Review by EriolEandur — CHANGES_REQUESTED** [review]
see 2 inline comments

File: .idea/jarRepositories.xml (hunk 1 above)
Comment (IoannisKl) [reply .idea/jarRepositories.xml:1]: Idea file is removed from the gothub repository.

File: src/main/java/me/John/BossBanner/Main.java (hunk 2 above)
Comment (IoannisKl) [reply src/main/java/me/John/BossBanner/Main.java]: Permissions are added as well as renamed.

File: .idea/jarRepositories.xml (hunk 1 above)
Comment (IoannisKl) [reply .idea/jarRepositories.xml]: Now the ./idea file is removed

**Review by EriolEandur — APPROVED** [review]

**Pull request merged**